### ISSUE TITLE
Scroll callback "direction" output suggestion

### ISSUE THREAD

**Issue by devenini** [darkroomengineering/lenis#48]
Hi! 

After using the scroll callback, I've been wondering if the returned `direction` parameter should output the direction of the user's scroll input instead of the global instance direction.

## Current behavior
```js
import Lenis from '@studio-freight/lenis'

const lenis = new Lenis({
  	direction: 'vertical', 
  	smooth: true,
})

lenis.on('scroll', ({ direction }) => {
  	console.log({direction}); // In that case, it'll always return `vertical` (lenis.direction)
})
```

## Suggested behavior
```js
import Lenis from '@studio-freight/lenis'

const lenis = new Lenis({
  	direction: 'vertical', 
	smooth: true,
})

lenis.on('scroll', ({ direction }) => {
  console.log({direction}); // returns `1` or `-1`. Based on user scroll event
})
```
Would return `1` or `-1` depending if the user is scrolling up and down (vertical) or left and right (horizontal).

I think this solution would make more sense since all other properties of that event are related to a scroll action, not the global instance.

## Workaround 
I've found a workaround to reproduce the suggested behavior. But I would like to know your thoughts on having this behavior built-in the library!
```js
import Lenis from '@studio-freight/lenis'

let scrollDirection = 1
let lastProgress = 0

const lenis = new Lenis({
	direction: 'vertical', 
 	smooth: true,
})

lenis.on('scroll', ({ progress }) => {
	//get direction
	if (progress > lastProgress) {
		if (scrollDirection != 1) {
			scrollDirection = 1;
		}
	} else {
		if (scrollDirection != -1) {
			scrollDirection = -1;
		}
	}
})
```

**Comment by arzafran** [comment]
hey! I've made a PR to add this functionality already, we're discussing if `direction` is semantic enough. would love to know your thoughts too. will land soon!

**Comment by arzafran** [comment]
landed in v0.2.20

**Comment by joshmeads** [comment]
Good improvement for sure. My workaround was simply checking if velocity was `<` or `>` 0.
